### PR TITLE
Russian language for installation folder

### PR DESCRIPTION
**Pull request by Kostelano**
Hi! Russian language for installation folder.

### CODE REVIEW

File: joomla_v4/translations/core/installation/language/ru-RU/langmetadata.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<metafile client="installation">
+	<name>Russian (Russia)</name>
+	<version>4.1.3</version>
```
Comment (tecpromotion) [inline joomla_v4/translations/core/installation/language/ru-RU/langmetadata.xml:4]:
```suggestion
	<version>4.1.3.1</version>

```

**Review by tecpromotion — CHANGES_REQUESTED** [review]
language version should be 4 digits

**Review by tecpromotion — APPROVED** [review]
thanks @Kostelano

**Comment by Kostelano** [comment]
Thanks too :)

**Pull request merged**